webapp/src/app/components/layers/layers.component.ts

```diff
@@ -152,6 +155,21 @@ export class LayersComponent implements OnInit {
 		this.selectedLayer.updateLevel(level);
 	}
 	
+	updateSize() {
+		this.selectedLayer?.resize(this.selectedLayer?.details.width, this.selectedLayer?.details.height);
```

Comment:
How I would implement it:
* Allow the user to change the values but it does not affect anything until they hit the refresh button. The refresh button should be highlighted when there are "unsaved" changes.
The other variant would be:
* Changing the values instantly changes the size but stores the cutoff tiles somewhere and if the size is increased again then those tiles are restored. This would also remove the need for the refresh button.